docs/normalization-rules.rst

```diff
@@ -90,6 +90,14 @@ string, it points to a :doc:`custom method <customize>`.
    >>> v.errors
    {'a': ["default value for 'a' cannot be set: Circular dependencies of default setters."]}
 
+You can even use both `default` and `readonly` on the same field. This will
```

Comment:
don't let this bit delay this pr to be merged, but a link do the other doc on each side would be handy.
other rule names are commonly enclosed by double backticks for monospaced appearance.
is the feature/fix worthenough to extend the docs?